docs/manifest.json

```diff
@@ -723,6 +723,18 @@
 					"title": "FAQs",
 					"description": "Miscellaneous FAQs from our community",
 					"path": "./tutorials/faqs.md"
+				},
+				{
+					"title": "Best practices",
+					"description": "Guides to help you make the most of your Coder experience",
+					"path": "./tutorials/best-practices/index.md",
```

Comment:
agreed - but future-proofing

I'm leaning towards just putting it where it would belong now. I'm also open to moving it up a level (outside of tutorials) - that'd make sense too

+ If it's cool to, we can edit the category/index.md to add a request for contributions. I'd love getting some community involvement here

Reply:
Future-proofing sounds good, down to leave this as-is.